src/data/demo-titles.ts

```diff
@@ -13,8 +13,8 @@ export const DEMO_DESCRIPTIONS: {[key: string]: string} = {
 };
 
 export const DEMO_MODEL_LINKS: {[key: string]: string} = {
-  resnet50:  'https://github.com/onnx/models/tree/master/resnet50',
-  squeezenet: 'https://github.com/onnx/models/tree/master/squeezenet',
+  resnet50:  'https://github.com/onnx/models/tree/master/models/image_classification/resnet',
```

Comment:
besides the links to https://github.com/onnx/models, do we have the links to the model file ( able to download directly )